### DISCUSSION TITLE
Freedom Planet 1

### DISCUSSION THREAD

**Discussion by ANoobAtGaming** [skyline-emu/skyline#1119] [Game Requests]
Freedom planet is a 2d platformer developed by GalaxyTrail. Plays out like sonic except more variety and combat, the quality is very similar to sonic mania as both are fast paced, 2d side-scrolling pixel sprite animated game. I suggest to make the game supported to skyline emulator as its quality is similarly simple yet complicated than sonic mania.

![images (2)](https://user-images.githubusercontent.com/103318023/165679790-e146e52c-99b9-43e3-a3de-023b0544050c.jpeg)
![images (3)](https://user-images.githubusercontent.com/103318023/165679821-10da8c72-d17f-4d97-af1b-e9eb164b2d56.jpeg)
![ss_23190eacb5008b1fcf55bac110daa675dfd5fb7e 600x338](https://user-images.githubusercontent.com/103318023/165679831-b2e2b02a-91a2-4fb3-9e7c-ed464c0980e2.jpg)
![images (4)](https://user-images.githubusercontent.com/103318023/165679914-e66e5d75-69c4-4749-b5e4-ee1e9abc9b18.jpeg)

**Comment by Selever19** [comment]
I hope it works in skylane
